README.rst

```diff
@@ -1,11 +1,16 @@
 Titanfall 2 + Northstar Modding Docs
 ====================================
 
-This repo contains the technical documentation on all things related to making Northstar
-mods.
+This repo contains documentation and guides on most things you need to know about creating mods for Northstar.
 
-Its currently in the early stages of development, so there might be some unpolished
-things about it.
+Especially the technical documentation is not done yet and might be incorrect or outdated in some places.
 
-If you have some knowledge you want to share, but don't want to write a PR, you can make
-a issue explaining what you know and we can try to integrate that into the docs.
+If you have some knowledge to share, but don't want to write a PR, you can make an issue explaining what you know and we can try to integrate that into the docs.
+
+Contributing
+------------
+
+The only prerequisites are python and pip. On Linux simply run `run.sh` in the root dir and in windows execute `run.ps1` with powershell in the root dir. You may need to change the Powershell execution policies in order to run the script.
```

Comment:
isnt ``pip`` always installed with python? 
Also we could add how to change the persmission in windows and or linux

Reply:
1. no
2. I think it's fair to expect someone to google how to run a script if that's the issue